src/site/_includes/components/BaseCard.js

```diff
@@ -32,11 +32,12 @@ const postTags = require('../../_data/postTags');
  */
 class BaseCard {
   constructor({post, featured = false, className = ''}) {
-    this.post = post;
+    this.element = post;
```

Comment:
I'm not sure I see the value in renaming `post` to `element`, especially if we're keeping the `{post}` argument in the constructor. I think it makes the code less obvious, IOW, it's introducing ambiguity but we don't gain anything from it. Essentially we know that 'post' is referring to a post item in a collection, and this abstracts that. I'd vote to just keep it as 'post'.

Reply:
I was actually thinking that the BaseCard should use `element` instead of `post` only because it ends up being the root/base for not just `PostCard` but also `AuthorCard`. So it thought a more generic name would be appropriate...